ci_scripts/download_localized_strings_from_lokalise.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+if [[ -z $(which lokalise2) ]]; then
+    echo "Installing lokalise2 via homebrew..."
+    brew tap lokalise/cli-2
+    brew install lokalise2
+fi
+
+if [[ -z $(which recode) ]]; then
+    echo "Installing recode via homebrew..."
+    brew install recode
+fi
+
+API_TOKEN=$(fetch-password mobile/lokalise/token -q)
+PROJECT_ID=$(fetch-password mobile/lokalise/ios -q)
+LANGUAGES="da,de,en-GB,en,es-AR,es,fi,fr-CA,fr,it,ja,ko,nb,nl,nn-NO,pt-BR,pt-PT,ru,sv,tr,zh-HANS,zh-HK,zh-TW"
+# Chinese (Traditional) is zh-TW by default in lokalise so we have to map it to zh-Hant
+LANG_MAP='[{"original_language_iso":"zh-TW","custom_language_iso":"zh-Hant"}]'
+
+lokalise2 --token $API_TOKEN \
+          --project-id $PROJECT_ID \
+          file download \
+          --format strings \
+          --filter-langs $LANGUAGES \
+          --language-mapping $LANG_MAP \
+          --export-sort "a_z" \
+          --directory-prefix "%LANG_ISO%.lproj" \
+          --original-filenames=true \
+          --unzip-to Stripe/Resources/Localizations/
```

Comment:
The workflow we have with translators has than go through a few steps, and the string is only complete when it's in the `Final` state. To pull only `Final` keys you can add
```
--custom-translation-status-ids final_status_id
```

ios sdk final status id is 587
ios 3ds2 sdk final status id is 583

Reply:
oh cool, I'll add!